content/millennium/r4/financial/general/charge-item.md

```diff
@@ -157,6 +157,65 @@ List an individual ChargeItem by its id:
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Operation: charge-item-credit
+
+<%= beta_tag(action: true) %>
+
+Creates an offsetting ChargeItem for an existing debit ChargeItem.
+
+    POST /ChargeItem/:id/$credit
+    
+_Implementation Notes_
+
+* Only the body fields mentioned below are supported. Unsupported fields will be ignored.    
+    
+### Authorization Types
+
+<%= authorization_types(provider: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', Accept: 'application/fhir+json', 'Content-Type': 'application/fhir+json'} %>
+
+### Body Fields
+
+Notes:
+
+* versionId, transactionId, subject and context are the required fields.
```

Comment:
Is this note really necessary since the Body fields table below has the Required Yes/No for each field?

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/726/commits/e08424a8d677108ea8d601e5d2219e41599cf259#diff-03f941e4ceb2839bfe0b49a5cc913e04919560e289603ae7b08b875f86ec568dL182-L185